test/e2e_node/remote/utils.go

```diff
@@ -43,6 +60,19 @@ func installCNI(host, workspace string) error {
 	if output, err := SSH(host, "sh", "-c", cmd); err != nil {
 		return fmt.Errorf("failed to install cni plugin on %q: %v output: %q", host, err, output)
 	}
+
+	// The added CNI network config is not needed for kubenet. It is only
+	// used when testing the CNI network plugin, but is added in both cases
+	// for consistency and simplicity.
+	glog.V(2).Infof("Adding CNI configuration on %q", host)
+	cniConfigPath := filepath.Join(workspace, cniConfDirectory)
+	cmd = getSSHCommand(" ; ",
+		fmt.Sprintf("mkdir -p %s", cniConfigPath),
+		fmt.Sprintf("echo %s > %s", quote(cniConfig), filepath.Join(cniConfigPath, "mynet.conf")),
```

Comment:
I didn't want to combine the CNI configuration with the CNI plugin binaries, I also thought it might be more obvious in the code as to what what the config is doing rather than refer to it by some name - that seemed more opaque to me.  If you have a strong preference to put in the tarball I can certainly do that.